Run `python3 dev/update_changelog.py --prev-version 2.1...` (#12423)
mlflow/mlflow: CHANGELOG.md

# CHANGELOG

## 2.14.1 (2024-06-20)

> MLflow 2.14.1 includes several major features and improvements

Reviewer: ```suggestion
MLflow 2.14.1 is a patch release that contains several bug fixes and documentation improvements
```

> Bug fixes:
> 
> - [Models] Fix params and model_config handling for llm/v1/xxx Transformers model (#12401, @B-Step62)
> - [UI] Fix dark mode user preference (#12386, @daniellok-db)

Reviewer: ```suggestion
Bug fixes:

- [Models] Fix params and model_config handling for llm/v1/xxx Transformers model (#12401, @B-Step62)
- [UI] Fix dark mode user preference (#12386, @daniellok-db)
- [Docker] Fix docker image failing to build with `install_mlflow=False` (#12388, @daniellok-db)
```

# CHANGELOG

## 2.14.1 (2024-06-20)

MLflow 2.14.1 is a patch release that contains several bug fixes and documentation improvements

Bug fixes:

- [Models] Fix params and model_config handling for llm/v1/xxx Transformers model (#12401, @B-Step62)
- [UI] Fix dark mode user preference (#12386, @daniellok-db)
- [Docker] Fix docker image failing to build with `install_mlflow=False` (#12388, @daniellok-db)

Documentation updates:

- [Docs] Add link to langchain autologging page in doc (#12398, @xq-yin)
- [Docs] Add documentation for Models from Code (#12381, @BenWilson2)

Small bug fixes and documentation updates:

> #12415, #12396, #12394, @harupy; #12403, #12382, @BenWilson2; #12397, @B-Step62; #12388, @daniellok-db

Reviewer: ```suggestion
#12415, #12396, #12394, @harupy; #12403, #12382, @BenWilson2; #12397, @B-Step62
```
